test/src/Token/AccessTokenTest.php

```diff
@@ -189,6 +189,28 @@ public function testInvalidExpiresIn()
         self::tearDownForBackwardsCompatibility();
     }
 
+    public function testInvalidExpires()
+    {
+        $options = [
+            'access_token' => 'access_token',
+            'expires' => 'TEXT',
+        ];
+
+        $token = $this->getAccessToken($options);
+
+        $this->assertSame($token->getExpires(), $token->getTimeNow());
```

Comment:
I'm not sure what you're trying to test here, but `'TEXT'` can't be the same as the return value of `getTimeNow()`, which is the current time stamp, so the tests are failing.